test/e2e/autoscaling/dns_autoscaling.go

```diff
@@ -237,13 +237,13 @@ func getExpectReplicasFuncLinear(c clientset.Interface, params *DNSParamsLinear)
 	return func(c clientset.Interface) int {
 		var replicasFromNodes float64
 		var replicasFromCores float64
-		nodes, err := e2enode.GetReadySchedulableNodes(c)
```

Comment:
The original GetReadySchedulableNodes() returns nodes which are filter with not only IsNodeSchedulable but also isNodeUntainted.
Don't we need to specify isNodeUntainted for this replacement?